treeherder/webapp/api/performance_serializers.py

```diff
@@ -282,7 +288,8 @@ class PerformanceQueryParamsSerializer(serializers.Serializer):
     parent_signature = serializers.CharField(required=False, allow_null=True, default=None)
     signature = serializers.CharField(required=False, allow_null=True, default=None)
     no_subtests = serializers.BooleanField(required=False)
-    all_data = serializers.BooleanField(required=False, default=False)
+    all_data = OptionalBooleanField()
```

Comment:
Do we still need a default value for `all_data`?

Reply:
Yes, it looks like we have some frontend code that's not providing this param explicitly ([example](https://github.com/mozilla/treeherder/blob/b24a8d0219d77ee51b1c5b23e8defea1eff6ea75/ui/perfherder/compare/CompareSubtestDistributionView.jsx#L118-L122)).